machine/delta/delta_tn48m/u-boot/series

```diff
@@ -0,0 +1,2 @@
+0001-Marvell-u-boot-armada7020-support.patch
```

Comment:
Why is this patch needed. The board should be supported by U-Boot since v2017.05. The patch is also too big and adds supports for all kinds of Marvell devices.

It also has several whitespace issues, which should be fixed.